public/app/core/components/sidemenu/BottomNavLinks.test.tsx

```diff
@@ -1,115 +0,0 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
-import { BrowserRouter } from 'react-router-dom';
-import appEvents from '../../app_events';
-import { ShowModalReactEvent } from '../../../types/events';
-import { HelpModal } from '../help/HelpModal';
-import BottomNavLinks from './BottomNavLinks';
-
-jest.mock('../../app_events', () => ({
-  publish: jest.fn(),
-}));
-
-describe('BottomNavLinks', () => {
-  const mockUser = {
-    id: 1,
-    isGrafanaAdmin: false,
-    isSignedIn: false,
-    orgCount: 2,
-    orgRole: '',
-    orgId: 1,
-    login: 'hello',
-    orgName: 'mockOrganization',
-    timezone: 'UTC',
-    helpFlags1: 1,
-    lightTheme: false,
-    hasEditPermissionInFolders: false,
-  };
-
-  it('renders the link text', () => {
-    const mockLink = {
-      text: 'Hello',
-    };
-
-    render(
-      <BrowserRouter>
-        <BottomNavLinks link={mockLink} user={mockUser} />
-      </BrowserRouter>
-    );
-    const linkText = screen.getByText(mockLink.text);
-    expect(linkText).toBeInTheDocument();
-  });
-
-  it('attaches the link url to the text if provided', () => {
-    const mockLink = {
-      text: 'Hello',
-      url: '/route',
-    };
-
-    render(
-      <BrowserRouter>
-        <BottomNavLinks link={mockLink} user={mockUser} />
-      </BrowserRouter>
-    );
-    const link = screen.getByRole('link', { name: mockLink.text });
-    expect(link).toBeInTheDocument();
-    expect(link).toHaveAttribute('href', mockLink.url);
-  });
-
-  it('creates the correct children for the help link', () => {
```

Comment:
🤔 Maybe we can add part of these tests cases on the new `BottomSection` component test? or we can create a test for the `getFooterLinks` function,  It seems we can benefit to check we are generating the correct items based on the permissions from backend? :)

Reply:
`BottomSection` was also the last place to convert to RTL so win-win! 🥳